frontends/jest.jsdom.config.ts

```diff
@@ -1,6 +1,8 @@
 import { resolve } from "path"
 import type { Config } from "@jest/types"
 
+process.env.SITE_NAME = "MIT Open"
```

Comment:
wondering if this can just fall back to the [default](https://github.com/mitodl/mit-open/pull/1121/files#diff-cf829b1ff5e1c8d9a75406c84d056628f3406931f3e539a198db34b85a7936e5R54) since I dont think we explicitly set environment variables even for testing purposes

Reply:
The idea for the tests is that we mock the process.env value by setting it explicitly here, then expect it in the tests, both hardcoded.

We could not hardcode, instead provide on environment, and then check e.g. ``expect(document.title).toBe(`About Us | ${process.env.SITE_NAME}`)`` in the tests, though not sure it adds anything above mocking the environment for testing.

Reply:
thats true. i raised it since i didnt see us setting an env variable directly elsewhere  - dont think its a blocker. approving.